clients/hadoopfs/pom.xml

```diff
@@ -287,7 +287,7 @@ To export to S3:
         <dependency>
             <groupId>io.lakefs</groupId>
             <artifactId>sdk</artifactId>
-            <version>1.12.0</version>
+            <version>1.14.0</version>
```

Comment:
Why +2?

Reply:
1.14 + contains all external principal API's. 
So it will end up 1.14.x with the login API merged.